macaroonbakery/checkers.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2017 Canonical Ltd.
+# Licensed under the LGPLv3, see LICENCE file for details.
+
+
+class Caveat:
+    '''Represents a truthful condition for a check to complete successfully.
+
+    If Location is non-empty, the caveat must be
```

Comment:
The capitalization seems like a carryover from Go.

    '''If location is provided, the caveat must be discharged by
    a third party at the given location (a URL string).

    The namespace parameter holds the namespace URI string of the
    condition - if it is provided, it will be converted to a namespace prefix
    before adding to the macaroon.
    '''

?

Also, maybe this might be better as a namedtuple rather than a class?